webpack.prod.js

```diff
@@ -0,0 +1,6 @@
+ const merge = require('webpack-merge');
+ const common = require('./webpack.common.js');
+
+ module.exports = merge(common, {
+   mode: 'development',
```

Comment:
Why `development`?

Reply:
Sorry that's a mistake, changing it now to 'production'